daemon/daemon_main.go

```diff
@@ -0,0 +1,1113 @@
+// Copyright 2016-2018 Authors of Cilium
```

Comment:
If this file was created, isn't a `daemon/main.go` missing? :thinking:

Reply:
new `daemon/main.go` is in this PR